nlp/berlin/options.go

```diff
@@ -0,0 +1,32 @@
+package berlin
+
+import (
+	"net/http"
+	"net/url"
+)
+
+const (
+	// List of available headers
+	Authorization string = "Authorization"
+	CollectionID  string = "Collection-Id"
+)
```

Comment:
what are you using these for? there is a headers/headers.go file which stores them.